.github/workflows/sonatypeDeployment.yml

```diff
@@ -46,3 +46,25 @@ jobs:
             ORG_GRADLE_PROJECT_signingKey: ${{ secrets.SIGNING_KEY }}
           run: ./gradlew clean publishToSonatype closeSonatypeStagingRepository --max-workers 1
 
+  publish_mono_release:
```

Comment:
Will likely have to integrate static artifact deploy into normal agent publishing job